### PR TITLE
Use posix archive format

### PR DESCRIPTION
**Pull request by smorimoto**
See #475 for details.

Closes: https://github.com/actions/cache/issues/326

### CODE REVIEW

**Comment by smorimoto** [comment]
@aiqiaoy This should now be in a mergeable state. Could you review it?

**Comment by smorimoto** [comment]
CC: @joshmgross

**Comment by smorimoto** [comment]
Related issue: https://github.com/actions/cache/issues/326

File: packages/cache/src/internal/tar.ts [hunk 1]
```diff
@@ -101,6 +101,7 @@ export async function createTar(
     }
   }
   const args = [
+    '--posix',
```
Comment (konradpabjan) [inline packages/cache/src/internal/tar.ts:104]: I looked over the previous PR, I _think_ this won't cause any compat issues. No objections from me 👍

File: packages/cache/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/cache",
-  "version": "1.0.1",
+  "version": "1.1.0",
```
Comment (konradpabjan) [inline packages/cache/package.json:3]: This is a pretty small change so I think a patch bump to version `1.0.2` is sufficient

Could you also update `package-lock.json`? There is a version number there as well.

Comment (smorimoto) [reply packages/cache/package.json:3]: Oh indeed.

Comment (smorimoto) [reply packages/cache/package.json]: It was changed with the latest commit.

File: packages/cache/src/internal/tar.ts (hunk 1 above)
Comment (smorimoto) [reply packages/cache/src/internal/tar.ts:104]: Yeah, this shouldn't cause any problems.

**Comment by smorimoto** [comment]
@konradpabjan Thanks for your review.

**Comment by aiqiaoy** [comment]
Yeah sorry I was on-call when I reviewed the original PR and completely forgot about following up. Looks good!

**Comment by konradpabjan** [comment]
Version `1.0.2` has been published: https://www.npmjs.com/package/@actions/cache/v/1.0.2

**Comment by smorimoto** [comment]
@konradpabjan Thanks again for your work :)

**Comment by smorimoto** [comment]
@aiqiaoy No worries! I sometimes do something similar to you.

**Pull request merged**